pkgs/by-name/co/comfyui/package.nix

```diff
@@ -0,0 +1,143 @@
+{ lib
+, python311
+, linkFarm
+, writers
+, writeTextFile
+, fetchFromGitHub
+, stdenv
+, symlinkJoin
+, config
+, gpuBackend ? (
+  if config.cudaSupport
+  then "cuda"
+  else if config.rocmSupport
+  then "rocm"
+  else "none"
+)
```

Comment:
This isn't used anywhere else, so Torch isn't getting compiled with CUDA and comfyui crashes at startup. Is there something blocking this?

Reply:
This may be vestigial, but I'm not sure.  I have older code doing things like using `torchWithCuda` if the `gpuBackend` is `"cuda"` for example.  I don't think that's right, because then you can't override your own `torch` (like via `torch-bin`).  This is a place where contributor documentation/advice could use some help.

In addition, I'm mostly blocked on `$TIME` and I can't even get this building presently because of `torch` via https://github.com/NixOS/nixpkgs/pull/268378#discussion_r1643737084.

Reply:
@wyndon you might try again since my latest force push.  It brings back in the stuff using `gpuBackend`.  I don't have it completely verified to work though due to what I believe are unrelated problems.

Reply:
I've worked past my issue and the fix does indeed work for me.